patterns/gitops/multi-cluster-hub-spoke-argocd/README.md

```diff
@@ -59,7 +59,20 @@ that temporary file with the `kubectl` configuration. This approach offers the
 advantage of not altering your existing `kubectl` context, allowing you to work
 in other terminal windows without interference.
 
-### Monitor GitOps Progress for Addons
+### Deploy ArgoCD Apps of ApplicationSets for Addons
+
+This command verifies the initial ArgoCD installation, ArgoCD will be re-configured when the addons are deployed and configured from git.
+```shell
+kubectl --context hub get all -n argocd
+```
+This command creates the application set manifest to deploy the addons.
+```shell
+kubectl --context hub apply -n argocd -f ../hub/bootstrap/addons.yaml
+```
```

Comment:
change to kubectl apply with Terraform

Reply:
@candonov I don't understand the comment
> change to kubectl apply with Terraform

In all our patterns now, we moved away from using terraform providers like kubernertes, helm, or kubectl and instead tell the user to use `kubectl apply` this way is easier to understand like we have in the single cluster gitops-bridge pattern here https://github.com/aws-ia/terraform-aws-eks-blueprints/tree/main/patterns/gitops/getting-started-argocd#deploy-the-addons